app/src/main/java/chat/rocket/android/fragment/chatroom/RoomFragment.java

```diff
@@ -295,9 +301,15 @@ private boolean closeSideMenuIfNeeded() {
   }
 
   private void setupMessageComposer() {
+    extraActionItems = new ArrayList<>(3); // fixed number as of now
+    extraActionItems.add(new ImageUploadActionItem());
+    extraActionItems.add(new AudioUploadActionItem());
+    extraActionItems.add(new VideoUploadActionItem());
```

Comment:
This is not used in MessageFormLayout. So it would be better to set in `showExtraActionSelectionDialog`.